packages/app/pages/organization/user/[uid]/edit.tsx

```diff
@@ -75,29 +75,37 @@ export default function Edit(): ReactElement {
     const { data: userData, error } = useSWR(
       `api/v1/organization/[oid]/get_user/[uid]`,
       async () =>
-        API.organizations
-          .getUser(organization?.id, Number(uid))
-          .then((userInfo) => {
-            userInfo.courses = userInfo?.courses.map((course) => {
-              return {
-                ...course,
-                // Needed for antd table to fill the keys with course id
-                key: course.id,
-              }
-            })
-            return userInfo
-          }),
+        API.organizations.getUser(organization?.id, uid).then((userInfo) => {
+          userInfo.courses = userInfo?.courses.map((course) => {
+            return {
+              ...course,
+              // Needed for antd table to fill the keys with course id
+              key: course.id,
+            }
+          })
+          return userInfo
+        }),
     )
 
     if (error) {
-      router.push('/organization/settings')
+      console.log('Error: ', error)
```

Comment:
Remove console.log?

Reply:
should be fine? according to lighthouse, it's best practice to log errors in the console.

Reply:
It's a bad practice to leave debug statements in productions